packages/web-client/app/components/card-pay/withdrawal-workflow/transaction-status/index.ts

```diff
@@ -0,0 +1,70 @@
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+import { WorkflowCardComponentArgs } from '@cardstack/web-client/models/workflow/workflow-card';
+import Layer1Network from '@cardstack/web-client/services/layer1-network';
+import Layer2Network from '@cardstack/web-client/services/layer2-network';
+import { currentNetworkDisplayInfo as c } from '@cardstack/web-client/utils/web3-strategies/network-display-info';
+
+import BN from 'bn.js';
+import { TransactionReceipt } from 'web3-core';
+
+class CardPayWithdrawalWorkflowTransactionStatusComponent extends Component<WorkflowCardComponentArgs> {
+  @service declare layer1Network: Layer1Network;
+  @service declare layer2Network: Layer2Network;
+
+  @tracked completedCount = 1;
+
+  constructor(owner: unknown, args: WorkflowCardComponentArgs) {
+    super(owner, args);
+    this.layer1Network
+      .awaitBridged(this.layer1BlockHeightBeforeBridging!)
+      .then((transactionReceipt: TransactionReceipt) => {
+        this.args.workflowSession.update(
+          'completedLayer1TransactionReceipt',
+          transactionReceipt
+        );
+        this.completedCount = 3;
+        this.args.onComplete();
+      });
+  }
+
+  get layer1BlockHeightBeforeBridging(): BN | undefined {
+    return this.args.workflowSession.state.layer1BlockHeightBeforeBridging;
+  }
+
+  get progressSteps() {
+    return [
+      {
+        title: `Withdraw tokens from ${c.layer2.fullName}`,
+      },
+      {
+        title: `Bridge tokens from ${c.layer2.fullName} to ${c.layer1.fullName}`,
+      },
+      {
+        title: `Release tokens on ${c.layer1.conversationalName}: ${c.layer2.shortName}`, // FIXME script says "DAI", not "xDai" 🤔
```

Comment:
The script specifies

> Release tokens on mainnet: DAI

while the current configuration in `network-display-info` would only produce this

> Release tokens on mainnet: xDai

if this departure from the script seems okay, I can remove this note!

Reply:
The tokens being released on mainnet would be DAI and not xDai, so I don't think the departure is OK.

Reply:
Yep, DAI here is referring to the token's symbol, not the network's name. We should use DAI.

Reply:
This should be obtained from the component's workflow session's `withdrawalToken` property.

Reply:
oh yes, of course, thank you both. I can use the `getUnbridgedSymbol` function I added for the withdrawal amount card 🎉

Reply:
fixed in 01a9c66b8